TESTS/baremetal.json

```diff
@@ -0,0 +1,8 @@
+{ 
+    "requires": [ 
```

Comment:
isn't this forcing baremetal for all tests?

Reply:
No untill passes the command line argument --app-config with baremetal.json